app/services/cms/providers/strapi/connection.rb

```diff
@@ -0,0 +1,17 @@
+module Cms
+  module Providers
+    module Strapi
+      class Connection
+        API_TOKEN = ENV["STRAPI_API_KEY"]
+        URL = ENV["STRAPI_URL"]
```

Comment:
standardrb-rails prefers it if you set envvars to a value in your application config and then read them as `Rails.config.strapi_api_key`